requirements.txt

```diff
@@ -2,3 +2,5 @@ pyyaml>=3.13
 peewee>=3.6.4
 boto3>=1.8.6
 lazy-import>=0.2.2
+google-api-python-client>=1.7.4
+google-cloud-storage>=1.12.0
```

Comment:
These dependencies should be added to `setup.py` as well, so they are installed together with the library when installing with `pip`.

Reply:
oh but isn't that why you were using lazy_module? Because then having these in `setup.py` will force non-gcp users to install the client as well. 

I can add an `extras_require` to `setup.py`, so users can do
`pip install lazydata["gcp"]` to install the gcp requirements.

Reply:
The primary reason for lazy_import was because boto3 has a really long load time, so that adds to startup time when using the `lazydata` CLI. 

I've just looked at these two libraries - the two of them with all the dependencies are about 4MB to download. So, wondering how much hassle it is to have to install them additionally vs having to download them on first install? What do you think?